scripts/run-circle.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e -x
```

Comment:
I prefer the slightly more verbose form:

```bash
set -o errexit
set -o xtrace
```

if only because I can never quite keep them straight in my head.